odk1-src/central-projects.rst

```diff
@@ -125,6 +125,33 @@ Managing Project App Users
 
 To manage App Users for a Project, you can navigate to the Project overview page, then click on the :guilabel:`App Users` tab under the Project name. For more information about creating, managing, and retiring Project App Users, please see :ref:`this section <central-users-app-overview>`.
 
+.. _central-projects-form-workflow:
+
+Managing Project Form Workflow
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Right now, Central offers two features around Form Workflow:
+
+ - Each Form's :ref:`Lifecycle State <central-forms-lifecycle>` controls whether App Users can download and/or submit to that Form. Near the end of a Form's life, for example, it makes sense to disallow downloading the Form as a blank, but still receive any submissions that have already been created.
+ - Access to download and submit each Form can be customized per App User associated with the Project. When first creating a Form, for example, it makes sense to only allow a testing user access to the Form so that one can be sure that it works before rolling it out to all users.
+
+In the future, additional workflow features are planned to make processes like training and drafting Forms easier.
+
+We place these workflow controls for all Forms in a single place, on the Form Workflow tab at the Project level. To access it, navigate to the Project and select the tab at the top of the page labelled :guilabel:`Form Workflow`.
+
+   .. image:: /img/central-projects/workflow.png
+
+On the left side of the Form Workflow page, you will find a list of all the Forms in the Project, along with a dropdown selection to set the lifecycle state for each one. Along the top, you will see all active App Users in the Project. At each row/column intersection, there is a checkbox that governs whether each App User is allowed access to each Form.
+
+If you are having trouble recalling what each Form State means, the :guilabel:`?` icon in the header will give you a quick recap:
+
+   .. image:: /img/central-projects/workflow-states.png
+
+As you make changes to Form States and App User access, they will be highlighted in yellow. You can make all the changes you'd like to apply at once (marking an old version of a Form as Closing while granting Open access to the new one, for example), and once you are satisfied with what you see you can click the Save button at the top-right of the screen to apply them all at once.
```

Comment:
I'd put the for example at the start of the parenthetical instead of at the end.